resource-managers/yarn/src/test/resources/META-INF/services/org.apache.spark.deploy.yarn.security.ServiceCredentialProvider

```diff
@@ -1 +0,0 @@
-org.apache.spark.deploy.yarn.security.TestCredentialProvider
```

Comment:
Which unit tests uses it?  Since the class doesn't exist anywhere, what effect does it have?